ZuluContent/SkillHandlers/Begging.cs

```diff
@@ -0,0 +1,180 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Scripts.Zulu.Engines.Classes;
+using Scripts.Zulu.Utilities;
+using Server.Misc;
+using Server.Targeting;
+using Server.Items;
+using Server.Network;
+using ZuluContent.Zulu.Skills;
+
+namespace Server.SkillHandlers
+{
+    public class Begging : BaseSkillHandler
+    {
+        private const int BegRange = 2;
+        private const int BegDelayMs = 2000;
+        private static readonly int BegCooldown = (int)TimeSpan.FromMinutes(30).TotalMilliseconds;
+        private static readonly Dictionary<uint, long> MobileBegCooldown = new();
+
+        private static readonly string[] BeggingLines =
+        {
+            "Give me something please...",
+            "I need something to eat!",
+            "I've got four children, please help me!",
+            "Could thee spare a dime?",
+            "Some thieves stole me everything, I'm broken now..."
+        };
+
+        private static readonly string[] ExitLines =
+        {
+            "I have to go!",
+            "I must leave!!",
+            "no....ehm...bye!",
+            "Bye!!",
+            "I must flee!!",
+            "I gotta go!",
+        };
+
+
+        private static readonly TargetOptions TargetOptions = new()
+        {
+            Range = 1,
+        };
+
+        public override SkillName Skill { get; } = SkillName.Begging;
+
+        public override async Task<TimeSpan> OnUse(Mobile from)
```

Comment:
A bit of a hybrid between the OSI begging and ZH3, used the important bits like gold calc etc and some of the iconic speech lines.

Reply:
ok cool